ios/Fabric/RNCPagerViewComponentView.mm

```diff
@@ -68,6 +75,14 @@ - (void)updateProps:(const facebook::react::Props::Shared &)props oldProps:(cons
         [_scrollView setScrollEnabled:newScreenProps.scrollEnabled];
     }
     
+    if (newScreenProps.animated != _animated) {
+        _animated = newScreenProps.animated;
+    }
+    
+    if (newScreenProps.page != _page && newScreenProps.page >= 0 && newScreenProps.page < _containerView.subviews.count) {
```

Comment:
Here we can check reactSubviews count, it would mostly do the same but be a bit more clear what are we checking here. But you can leave it like so.

Reply:
Changed it in `RNVViewPager.m`, because there is no `reactSubiews` in fabric implementation